cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/vis_builder/migration.spec.js

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  BASE_PATH,
+  toTestId,
+  VB_APP_PATH,
+  VB_INDEX_ID,
+  VB_METRIC_VIS_TITLE,
+  VB_PATH_INDEX_DATA,
+  VISUALIZATION_PATH_SO_DATA,
+} from '../../../../../utils/constants';
+import { CURRENT_TENANT } from '../../../../../utils/commands';
+
+if (Cypress.env('VISBUILDER_ENABLED')) {
+  describe('Vis Builder: Metric Chart', () => {
+    before(() => {
+      CURRENT_TENANT.newTenant = 'global';
+      cy.fleshTenantSettings();
+      cy.deleteIndex(VB_INDEX_ID);
+      cy.bulkUploadDocs(VB_PATH_INDEX_DATA);
```

Comment:
@LDrago27, i forget what was the suggestion with delete and insert? is this safe for runs with frequent write/deletes?